src/applications/disability-benefits/all-claims/constants.js

```diff
@@ -223,6 +223,8 @@ export const PTSD_CHANGE_LABELS = {
 export const ATTACHMENT_KEYS = [
   'form781Upload',
   'form781aUpload',
+  'form8940Upload',
+  'uploaded4192',
```

Comment:
This key may not be consistent with the others e.g. `form4192Upload`

Reply:
@zurbergram ^^

Reply:
changing